packaging/shark-platform/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "shark-platform"
+authors = [
+  {name = "SHARK Authors"},
+]
+description = "SHARK Platform meta package"
+readme = "README.md"
+license = {text = "Apache-2.0"}
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+]
+requires-python = ">= 3.10"
+
+version = "2.9.0"
```

Comment:
Could source this from a file, so the build_packages workflow can update a `_rc` version and then a single script like https://github.com/iree-org/iree/blob/main/build_tools/python_deploy/promote_whl_from_rc_to_final.py can run on all the packages when ready to promote and push to pypi.

Reply:
Sure, that can definitely be done. I think if we automatically build it, we also want to auto-generate a `requirements.txt` file which pulls in the correct versions (which means the exact `X.Y.ZrcYYYYMMDD` versions for a nightly meta-package` and the `Z.Y.*` packages for a stable release). WDYT?

Reply:
How about we start with manually authored and run. We can iterate on the automation after a few runs through the process.

Reply:
I added `build_tools/python_deploy/write_requirements.py` to write the file based on what version is provided via a previously written `version_local.json` (which we need to generate anyway).